### PR TITLE
No git

### LINKED ISSUE #216: git stage all files during initialising a repo?
I wouldn't normally prefer having an automated commit since the user might think it as a "magic".

More over, seeing from a user point of view, I wouldn't really like to commit right after creating a repo.

Leaving this with tagged discussion.

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #216, 
This will put --git in default state of `false` so that when user says --git he knows that there will some git activity, and we wont do aut commits without his knowledge

### CODE REVIEW

File: src/commands/create.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default asyncCommand({
 		git: {
 			description: 'Initialize version control using git',
 			type: 'boolean',
-			default: true
+			default: false
```
Comment (reznord) [inline src/commands/create.js]: I'm not really sure if setting `--git` to false by default or not. 

Leaving `--git` to `true` by default and not making an initial commit makes sense to me.

@developit any thoughts?

Comment (thangngoc89) [inline src/commands/create.js]: I think we are trying to hard here. User know how to initilize a git repo. Just leave it to user.

**Comment by prateekbh** [comment]
ok, 
we have 2 options here
*Option A:* leave `--git` to true and and not make the final commit.

*Option B:* leave `--git` to false and leave everything to user.

@thangngoc89 @reznord which one do you think guys?

**Comment by reznord** [comment]
@prateekbh I'm good with both, anyways for _Option B_ it is just a matter of one command away. So, I'm with @thangngoc89 here.

**Pull request merged**